concourse/scripts/environment.sh

```diff
@@ -9,38 +9,22 @@ hashed_password() {
   echo "$1" | shasum -a 256 | base64 | head -c 32
 }
 
-DEPLOY_ENV=${DEPLOY_ENV:-}
+DEPLOY_ENV=${1:-${DEPLOY_ENV:-}}
```

Comment:
Why is this positional parameter being re-introduced? I removed this in 5a70bcd because it's not used anywhere...